pkg/workspace/inference/preset_inferences.go

```diff
@@ -127,7 +129,7 @@ func GenerateModelWeightsCacheVolume(ctx context.Context, workspaceObj *v1beta1.
 }
 
 func GeneratePresetInference(ctx context.Context, workspaceObj *v1beta1.Workspace, revisionNum string,
```

Comment:
Please don't pass `configMap` from outside. Use `EnsureConfigOrCopyFromDefault` to get the configmap you want.